spec/data-model/README.md

```diff
@@ -276,15 +270,15 @@ A `Markup` object has a `kind` of either `"open"`, `"standalone"`, or `"close"`,
 each corresponding to _open_, _standalone_, and _close_ _markup_.
 The `name` in these does not include the starting sigils `#` and `/` 
 or the ending sigil `/`.
-The optional `options` for markup use the same `Option` as `FunctionAnnotation`.
+The optional `options` for markup use the same key-value mapping as `FunctionAnnotation`.
```

Comment:
"optional options" seems redundant? Consider making the text here exactly the same as in `FunctionAnnotation`

Reply:
```suggestion
The `options` for markup use the same key-value mapping as `FunctionAnnotation`.
```